meilisearch/models/task.py

```diff
@@ -11,7 +11,8 @@ class Task(CamelBase):
     status: str
     type: str
     details: Dict[str, Any]
```

Comment:
In another PR, this should be changed to `Union[Dict[str, Any], None]`

Reply:
https://github.com/meilisearch/meilisearch-python/issues/602